react/features/notifications/actions.ts

```diff
@@ -94,14 +94,13 @@ export function setNotificationsEnabled(enabled: boolean) {
  * Queues an error notification for display.
  *
  * @param {Object} props - The props needed to show the notification component.
- * @param {string} type - Notification type.
  * @returns {Object}
  */
-export function showErrorNotification(props: INotificationProps, type?: string) {
```

Comment:
Why not leave the type argument but put a default value `NOTIFICATION_TIMEOUT_TYPE.STICKY`?

Reply:
Agreed, leaves the flexibility there.

Reply:
> Why not leave the type argument but put a default value `NOTIFICATION_TIMEOUT_TYPE.STICKY`?

Will, do thanks.

Reply:
Done